frontend/src/Editor/Components/Table/Table.jsx

```diff
@@ -760,6 +772,7 @@ export function Table({
             ),
             Cell: ({ row }) => (
               <div className="d-flex flex-column align-items-center">
+                ]
```

Comment:
@stepinfwd Dude what's this 😅 

<img width="103" alt="image" src="https://user-images.githubusercontent.com/20333972/155484731-0bbb32f9-7c09-48e5-83c0-03295909b61c.png">

Reply:
Other than this I don't see any issue, remove this and merge the latest develop. And then let's merge it.

Reply:
updated